src/main.rs

```diff
@@ -220,6 +226,26 @@ fn quickadd(matches: &ArgMatches, text: String) -> Result<String, String> {
     Ok(projects::green_string("✓"))
 }
 
+#[cfg(not(tarpaulin_include))]
+fn task_edit(matches: &ArgMatches) -> Result<String, String> {
+    let config = fetch_config(matches)?;
+    let project_name = fetch_project(matches, &config)?;
```

Comment:
I see that there is a lot of work with the project entity. 

I might be wrong but isn't we now have some struct for projects?
@alanvardy I think it was one of the recent refactors, maybe too recent so it is not present here =)

Maybe we can fetch the project object, and from that point work with a strongly typed entity? Rather than use associated methods n times over and over.

I thought that we have the ability now to acquire Project objects. Then we can just use fields that are populated with related data (project.name, project.id, etc)

It is a minor thing and may be addressed in the future.
Other than that it is a good addition to the interface. =)

Reply:
I haven't done that yet, but would like to in another refactor. Great idea!